### ISSUE TITLE
investigate amp-3d-gltf tests timing out

### ISSUE THREAD

**Issue by cathyxz** [ampproject/amphtml#16080]
So far 2 out of 3 tests on amp-3d-gltf flake in our builds with a timeout. We should possibly see if there's something weird going on with the component itself.

**Comment by aghassemi** [comment]
interesting, it is such as simple test. If it is flaky, we should see what the root cause is which might be test utils/infra

**Comment by aghassemi** [comment]
(also seems to be passing on Master, is it failing in PRs only?)

**Comment by cathyxz** [comment]
I've seen it flake on two of my recent PRs quite a bit. It's always a timeout failure.

**Comment by aghassemi** [comment]
feel free to skip, it is not testing much anyway.

**Comment by aghassemi** [comment]
Closing for now since skipped with #16084

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile.  Do you have any updates?

**Comment by aghassemi** [comment]
too old, closing (we can easily find skipped tests in code anyway)
